stdlib/2/collections.pyi

```diff
@@ -18,7 +18,7 @@ _KT = TypeVar('_KT')
 _VT = TypeVar('_VT')
 
 # namedtuple is special-cased in the type checker; the initializer is ignored.
-def namedtuple(typename: str, field_names: Union[str, Iterable[Any]], *,
+def namedtuple(typename: AnyStr, field_names: Union[AnyStr, Iterable[Any]], *,
```

Comment:
As mentioned in my comment above, all or most of the `AnyStr` types in this PR should be `Union[str, unicode]` instead. Can you update them? If you are unsure about some of them I can help.